assets/src/apps/delivery/components/ActivityRenderer.tsx

```diff
@@ -404,13 +402,7 @@ const ActivityRenderer: React.FC<ActivityRendererProps> = ({
           target = ownerActivity ? `${ownerActivity.id}|${target}` : `${target}`;
         }
       }
-      const originalValue = snapshot[target];
```

Comment:
We don't need to do this check as we are fetching values from snapshot

Reply:
Curious, do you have any idea why it would behave differently between a `localhost` open+free vs a `Tokamak` open+free?

Reply:
I am not exactly sure about that but may be it due to the some timing issue in the sequence of things when they are executed.  Still I think it should have any impact in open and free.

The only thing is that we have to be very patience when testing the Open & free for Small world like sims as they are very heavy and some times after clicking, we have to wait till it completes the check.